### ISSUE TITLE
Document: Select

### ISSUE THREAD

**Issue by jma7** [vatlab/vat-docs#19]
When use hg38 as primary build, can't down load dbNSFP.

```
2.2 Select variants using annotation fields
Examples: Select variants by values of annotation fields

vtools use dbNSFP
INFO: Choosing version dbNSFP-hg38_3_5a from 10 available databases.
INFO: Downloading annotation database annoDB/dbNSFP-hg38_3_5a.ann
INFO: Downloading annotation database from anno/dbNSFP-hg38_3_5a.DB.gz
ERROR: Failed to download anno/dbNSFP-hg38_3_5a.DB.gz because it is not in the variant tools online repository or local stash directories.
```

**Comment by jma7** [comment]
It seems dbNSFP v3.5a is available at ```ftp://dbnsfp.softgenetics.com/```. Do you know anything about this file? @HenryLeongStat

**Comment by HenryLeongStat** [comment]
Yes, I think we are using it for the annotation now? The annotation database was updated last year if I remember correctly.

https://github.com/vatlab/VariantTools/blob/4bb667911ca7f688bbcbeed4e538b3756706a162/annotation/dbNSFP-hg38_3_5a.ann

**Comment by jma7** [comment]
So this file is not on the online repository?

**Comment by HenryLeongStat** [comment]
The `.ann` file is on the [vatlab/VariantTools](https://github.com/vatlab/VariantTools) repo.

The full path is [vatlab/VariantTools/annotation/dbNSFP_gene-3_5a.ann](https://github.com/vatlab/VariantTools/blob/4bb667911ca7f688bbcbeed4e538b3756706a162/annotation/dbNSFP_gene-3_5a.ann)

Also the built database is on the server that I used. (The name of that one ends with `04`?)

**Comment by BoPeng** [comment]
Try to use

```
wget http://bioinformatics.mdanderson.org/Software/VariantTools/repository/annoDB/dbNSFP-hg38_3_5a.DB.gz
```
and see if the file can be downloaded. Put the file to `~/.variant_tools/annoDB`, decompress and it should be usable.

If the database is OK, check the download mechanism and see if we can use `wget` instead of Python's download function.

**Comment by HenryLeongStat** [comment]
Need to try again.

**Comment by jma7** [comment]
The direct_url in ```dbNSFP-hg38_3_5a.ann``` should be
```direct_url=annoDB/dbNSFP-hg38_3_5a.DB.gz```, right now the link is ```anno``` instead of ```annoDB```.  Need to make changes on repository? @HenryLeongStat @BoPeng

**Comment by HenryLeongStat** [comment]
The direct_url of `dbNSFP-hg38_3_5a.ann` on github is empty. I guess the one being used is at one of the MDACC servers? 

@BoPeng Professor, I remember you use a script to add the direct_url (or md5??) to all the ann files.

**Comment by jma7** [comment]
https://github.com/vatlab/VariantTools/blob/v3_genoinfo/annotation/dbNSFP-hg38_3_5a.ann
@HenryLeongStat

**Comment by HenryLeongStat** [comment]
That is my bad... 😅

**Comment by jma7** [comment]
It seems to me it is using a version from repository. I updated the ann file on my local branch, but I am still getting the same error. Anybody could update the version on repository? @BoPeng
